core/src/autogluon/core/models/ensemble/fold_fitting_strategy.py

```diff
@@ -708,3 +733,31 @@ def _parse_ray_error(self, e):
             logger.warning(default_error_msg)
             e = NotEnoughCudaMemoryError
         return e
+    
+    def _sync_model_artifact(self, local_path, model_sync_path):
+        pass
```

Comment:
docstring + type hint. Do we even want to allow calling this in the scenarios it doesn't do anything?

Reply:
This is mainly serving as a general interface so that subclass can provide its own implementation. In the case of no syncing is needed, I think it makes sense to just leave an empty implementation there.